br/pkg/lightning/importer/get_pre_info.go

```diff
@@ -135,11 +140,16 @@ func NewTargetInfoGetterImpl(
 		return nil, errors.Trace(err)
 	}
 	var backendTargetInfoGetter backend.TargetInfoGetter
+	var pdCli pd.Client
 	switch cfg.TikvImporter.Backend {
 	case config.BackendTiDB:
 		backendTargetInfoGetter = tidb.NewTargetInfoGetter(targetDB)
 	case config.BackendLocal:
-		backendTargetInfoGetter = local.NewTargetInfoGetter(tls, targetDB, cfg.TiDB.PdAddr)
+		pdCli, err := pd.NewClientWithContext(ctx, []string{cfg.TiDB.PdAddr}, tls.ToPDSecurityOption())
```

Comment:
not closed, this part seems used in df service

Reply:
Yes. It's not used in lightning. df service need to close it if they update lightning dependencies.

Reply:
`pdCli` is shadowed inside, so df service cannot close it.

Reply:
cc @okJiang @dsdashun

Reply:
What about use `pdCli` as a variable for `NewTargetInfoGetterImpl`. The user who uses this function should handle pdCli by themselves

Reply:
I have seen `TargetInfoGetter.Close()`. I think it meet our demand, does it?